README.md

```diff
@@ -43,24 +48,106 @@ BBLAYERS += "${BSPDIR}/sources/meta-cyclonedx"
 
 ```sh
 INHERIT += "cyclonedx-export"
 ```
 
+### CycloneDX Specification Version
+
+By default, meta-cyclonedx generates **CycloneDX 1.6** format SBOMs. If you need
+compatibility with tools that only support CycloneDX 1.4, you can configure:
+
+```sh
+CYCLONEDX_SPEC_VERSION = "1.4"
+```
+
+**Version differences:**
+
+- **1.4**: Legacy format for compatibility with older tools
+- **1.6**: Modern format with enhanced metadata and timestamps (default)
+
+### Runtime vs Build-time Packages
+
 By default, meta-cyclonedx will only include run-time packages in the SBOM,
 which drastically reduces the number of potentially irrelevant packages.
 However, this can lead to valid packages being omitted from the SBOM
 (see [here](#potentially-missing-packages-after-runtime-filtering)).
+
 If preferred, you can add the following configuration setting
 (e.g in your local.conf), which will cause meta-cyclonedx to include
 all build-time packages as well:
 
-```
+```sh
 CYCLONEDX_RUNTIME_PACKAGES_ONLY = "0"
 ```
 
+### Component Scopes
+
+When including both runtime and build-time packages, meta-cyclonedx uses
+[CycloneDX component scopes](https://cyclonedx.org/docs/1.6/json/#components_items_scope)
+to differentiate between them:
+
+- Runtime packages are marked with `"scope": "required"`
+- Build-time only packages are marked with `"scope": "optional"`
+
+This allows tools to filter components based on their use case:
+
+- **CVE matching**: Focus on components with `"scope": "required"`
+- **License compliance**: Include all components regardless of scope
+- **Supply chain tracking**: Include all components regardless of scope
+
+Component scopes are enabled by default and available in both CycloneDX 1.4 and 1.6
+specifications. If you need to disable them (e.g., for compatibility with certain
+SBOM profiles or tools):
+
+```sh
+CYCLONEDX_ADD_COMPONENT_SCOPES = "0"
+```
+
+### Vulnerability Analysis Timestamps
+
+By default, vulnerability analysis records include `firstIssued` and `lastUpdated`
+timestamps when using CycloneDX 1.6. To generate minimal VEX documents without timestamps:
+
+```sh
+CYCLONEDX_ADD_VULN_TIMESTAMPS = "0"
+```
+
+### Advanced Configuration Summary
+
+```sh
+# Specification version (default: "1.6")
+CYCLONEDX_SPEC_VERSION = "1.6"  # or "1.4"
+
+# Include build-time packages (default: "1" = runtime only)
+CYCLONEDX_RUNTIME_PACKAGES_ONLY = "1"
+
+# Add component scopes (default: "1")
+CYCLONEDX_ADD_COMPONENT_SCOPES = "1"
+
+# Add vulnerability timestamps in 1.6 (default: "1")
+CYCLONEDX_ADD_VULN_TIMESTAMPS = "1"
+```
+
+### SBOM Profile Compatibility
+
+Different organizations may have specific SBOM profile requirements. The configuration
+options above allow you to customize the generated SBOMs to meet various needs:
+
+**If your tools or profiles don't support component scopes:**
+
+```sh
+CYCLONEDX_ADD_COMPONENT_SCOPES = "0"
+```
+
+**For legacy tool compatibility:**
+
+```sh
+CYCLONEDX_SPEC_VERSION = "1.4"
+```
+
```

Comment:
I'd suggest we remove this for now, for the same reasons as in https://github.com/iris-GmbH/meta-cyclonedx/pull/31#discussion_r2459858337.

If we add a section on SBOM profile compatibility, then it should address all feature flags, not just component scopes and spec version.